Seminars/Homeworks/Homework_003/Task_003.py

```diff
@@ -0,0 +1,7 @@
+# Задайте список из вещественных чисел. Напишите программу, которая найдёт разницу между максимальным и минимальным значением дробной части элементов.
+# Пример:
+# - [1.1, 1.2, 3.1, 5, 10.01] => 0.19
+
+lst = [1.1, 1.2, 3.1, 5, 10.01]
+new_lst = [round(i%1,2) for i in lst if i%1 != 0]
```

Comment:
```suggestion
new_lst = [round(i % 1, 2) for i in lst if i % 1 != 0]
```
Не забывайте пожалуйста, про пробелы